other_projects/cxxopts/cmake/cxxopts.cmake

```diff
@@ -159,5 +159,14 @@ function(cxxopts_install_logic)
             DESTINATION "${CMAKE_INSTALL_LIBDIR_ARCHIND}/pkgconfig"
     )
 
+    # this should fix directory permission derived from Travis-CI ubuntu ${BUILD_DIR} from 0775 to 0755.
```

Comment:
if it fixes, change to `fix` instead of `should` and reference the github issue too (#827)
(when travis ci is completed, the rpm pacakge should be updated in the release page as a pre-release version [4.3.x] or something)